x-pack/plugins/timelines/common/utils/field_formatters.test.ts

```diff
@@ -7,15 +7,29 @@
 
 import { eventDetailsFormattedFields, eventHit } from '@kbn/securitysolution-t-grid';
 import { EventHit } from '../search_strategy';
-import { getDataFromFieldsHits, getDataSafety } from './field_formatters';
+import { getDataFromFieldsHits } from './field_formatters';
 
 describe('Events Details Helpers', () => {
   const fields: EventHit['fields'] = eventHit.fields;
   const resultFields = eventDetailsFormattedFields;
   describe('#getDataFromFieldsHits', () => {
     it('happy path', () => {
       const result = getDataFromFieldsHits(fields);
-      expect(result).toEqual(resultFields);
+      const resultCategories = result.map((r) => r.category);
+      const expectedCategories = resultFields.map((r) => r.category);
+      const resultFieldsNames = result.map((r) => r.field);
+      const expectedFieldsNames = resultFields.map((r) => r.field);
+      const resultIsObjectArray = result.map((r) => r.isObjectArray);
+      const expectedIsObjectArray = resultFields.map((r) => r.isObjectArray);
+      const resultOriginalValue = result.map((r) => r.originalValue);
+      const expectedOriginalValue = resultFields.map((r) => r.originalValue);
+      const resultValues = result.map((r) => r.values);
+      const expectedValues = resultFields.map((r) => r.values);
+      expect(resultCategories).toEqual(expect.arrayContaining(expectedCategories));
```

Comment:
Is it still changing the sequence order of the array?

Reply:
No, and i just backed that change out, I was seeing the duplicated security solution test failing and misinterpreted the results, not realizing there were duplicates ha. Removed in c02920c3cc240bdf7ffa9a5723201011c7babf6a